sqlx-mysql/src/protocol/text/column.rs

```diff
@@ -62,6 +63,10 @@ bitflags! {
 }
 
 // https://dev.mysql.com/doc/internals/en/com-query-response.html#column-type
+// dead link ^
+// https://dev.mysql.com/doc/dev/mysql-server/9.0.0/page_protocol_com_query_response_text_resultset_column_definition.html
+// the "type of the column as defined in enum_field_types"
+// https://dev.mysql.com/doc/dev/mysql-server/9.0.0/field__types_8h.html
```

Comment:
feel free to suggest which of these links we should keep - the original link here was dead when i tried it.